ui/src/controller/track_decider.ts

```diff
@@ -101,11 +101,11 @@ const NETWORK_TRACK_REGEX = new RegExp('^.* (Received|Transmitted)( KB)?$');
 const NETWORK_TRACK_GROUP = 'Networking';
 const ENTITY_RESIDENCY_REGEX = new RegExp('^Entity residency:');
 const ENTITY_RESIDENCY_GROUP = 'Entity residency';
-const BATTERY_TRACK_REGEX = new RegExp('^batt\\..*$');
+const BATTERY_TRACK_REGEX = new RegExp('^(batt|power)\\..*$');
 const BATTERY_TRACK_GROUP = 'Battery';
 const GPU_QUEUES_REGEX = new RegExp('^(Universal|Timer|DMA|Compute)( IB)?$');
 const GPU_QUEUES_GROUP = 'GPU Queues';
-
+const globalCounters = ['Buffers', 'Cached', 'MemFree', 'MemTotal', 'SwapCached'];
```

Comment:
Why are these referred to as "global" when they are in a group called `Memory Usage`?